docs/get-started/how-to/poh-api.mdx

```diff
@@ -119,3 +100,78 @@ Parameters:
+
+Typical flow:
+1.	The user connects their Ethereum wallet.
+2.	Your app asks them to sign a message similar to:
+
+```
+{
+   "walletAddress":"0x9F912e81DC869aA66E1f29e0500A76ad12aCdc2d",
+   "signatureMessage":"I confirm that I am the owner of this wallet and consent to performing a risk assessment and issuing a Verax attestation to this address.",
+   "signInMessage":"in.sumsub.com wants you to sign in with your Ethereum account:\n0x9F912e81DC869aA66E1f29e0500A76ad12aCdc2d\n\nI confirm that I am the owner of this wallet and consent to performing a risk assessment and issuing a Verax attestation to this address.\n\nURI: https://in.sumsub.com\nVersion: 1\nChain ID: 59144\nNonce: WwqZknqSgNHEJPRChKNyzq\nIssued At: 2025-10-01T14:16:37.790Z",
+   "signature":"0xf989ef76c0d465f1f23b133b6dc7c261e76d6390853fed44f3b65416a29e00a841763733e1a556013d6e3be0421d52c8f74eb6c48bc16348e9e445899b1fb26c1c",
+   "walletProvider":"reown",
+   "walletName":"Trust Wallet",
+   "nonce":"WwqZknqSgNHEJPRChKNyzq"
+}
+```
+	3.	You generate a link including this signed payload and redirect or open it in an iframe:
+
+```
+https://in.sumsub.com/websdk/p/uni_BKWTkQpZ2EqnGoY7?msg=eyJ3YWxsZXRBZGRyZXNzIjoiMHg5RjkxMmU4MURDODY5YUE2NkUxZjI5ZTA1MDBBNzZhZDEyYUNkYzJkIiwic2lnbmF0dXJlTWVzc2FnZSI6IkkgY29uZmlybSB0aGF0IEkgYW0gdGhlIG93bmVyIG9mIHRoaXMgd2FsbGV0IGFuZCBjb25zZW50IHRvIHBlcmZvcm1pbmcgYSByaXNrIGFzc2Vzc21lbnQgYW5kIGlzc3VpbmcgYSBWZXJheCBhdHRlc3RhdGlvbiB0byB0aGlzIGFkZHJlc3MuIiwic2lnbkluTWVzc2FnZSI6ImluLnN1bXN1Yi5jb20gd2FudHMgeW91IHRvIHNpZ24gaW4gd2l0aCB5b3VyIEV0aGVyZXVtIGFjY291bnQ6XG4weDlGOTEyZTgxREM4NjlhQTY2RTFmMjllMDUwMEE3NmFkMTJhQ2RjMmRcblxuSSBjb25maXJtIHRoYXQgSSBhbSB0aGUgb3duZXIgb2YgdGhpcyB3YWxsZXQgYW5kIGNvbnNlbnQgdG8gcGVyZm9ybWluZyBhIHJpc2sgYXNzZXNzbWVudCBhbmQgaXNzdWluZyBhIFZlcmF4IGF0dGVzdGF0aW9uIHRvIHRoaXMgYWRkcmVzcy5cblxuVVJJOiBodHRwczovL2luLnN1bXN1Yi5jb21cblZlcnNpb246IDFcbkNoYWluIElEOiA1OTE0NFxuTm9uY2U6IFd3cVprbnFTZ05IRUpQUkNoS055enFcbklzc3VlZCBBdDogMjAyNS0xMC0wMVQxNDoxNjozNy43OTBaIiwic2lnbmF0dXJlIjoiMHhmOTg5ZWY3NmMwZDQ2NWYxZjIzYjEzM2I2ZGM3YzI2MWU3NmQ2MzkwODUzZmVkNDRmM2I2NTQxNmEyOWUwMGE4NDE3NjM3MzNlMWE1NTYwMTNkNmUzYmUwNDIxZDUyYzhmNzRlYjZjNDhiYzE2MzQ4ZTllNDQ1ODk5YjFmYjI2YzFjIiwid2FsbGV0UHJvdmlkZXIiOiJyZW93biIsIndhbGxldE5hbWUiOiJUcnVzdCBXYWxsZXQiLCJub25jZSI6Ild3cVprbnFTZ05IRUpQUkNoS055enEifQ==
+```
+
+	4.	The Sumsub verification process happens on their side.
+	5.	Once verified, the user is redirected back to your webapp, or your frontend receives a message event from the iframe wrapper.
+	6.	Optional: You can then query the Linea PoH API to confirm that the attestation has been issued onchain.
+
+#### Notes
+- The attestation is issued only if none exists or if the previous one has expired.
+- This method offers the best user experience and ensures the attestation is always up to date.
+- An alternative, with a slightly degrated UX, is to redirect the user to the Linea Hub PoH page (not live yet), to make them complete the verification there.
```

Comment:
what's the intended meaning of "degrated"? Seems like something like: 
- "more convoluted UX" 
- "lengthier UX flow" 
- "with an additional UX step"